W-21917743 apim relaunch may 11 jh (#355)
mulesoft/docs-exchange: modules/ROOT/pages/adding-amazon-gateway-scanner.adoc

= Adding a Scanner for Amazon API Gateway
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Add a scanner to discover, import, and sync agents from Amazon API Gateway into Exchange. Then you can govern the agents and consume them in other applications.

Reviewer: @JennyHajee 
```suggestion
Add a scanner to discover, import, and sync services from Amazon API Gateway into Exchange. Once imported, these services are available for governance, policy enforcement, and exposure through your developer portal.
```

> Add a scanner to discover, import, and sync services from Amazon API Gateway into Anypoint Exchange. Once imported, you can apply governance, enforce policies, and publish them to your developer portal.

Reviewer: ```suggestion
Add a scanner to discover, import, and sync APIs from Amazon API Gateway into Anypoint Exchange. Once imported, you can apply governance rulesets and publish them to your developer portal.
```

== Before You Begin
Before adding the scanner, verify that you have the permission, context, and these Amazon API Gateway credentials:

* Exchange Administrator permission 
* AWS APIGatewayScannerReadOnly permission

> * Knowledge about the Amazon Web Services region where the agents are deployed

Reviewer: @JennyHajee 
```suggestion
* Knowledge about the Amazon Web Services region where the services are deployed
```

* AWS access key ID
* AWS secret access key

== Add a Scanner for Amazon API Gateway
. Verify that you are in the business group where you want to add the scanner.
. From the sidebar in Exchange, click *Scanners*.
. Enter a name for the scanner.
. From *Scanner Run Configuration*, complete these fields or options:
+
[%header,cols="1,4"]
|===
|*Field/Option* |*Value*
|*Run Schedule* |Select a frequency and local time.
|*Sync Review* |Select an option: Auto-resolve or Ask to review.
|===

. From *Connection Configuration*, complete these fields:
+
[%header,cols="1,4"]
|===
|*Field* |*Value*
|*Provider* |Select *Amazon*.
|*Platform* |Select *API Gateway*.

> |*Service Type* |Select *MCPs*.

Reviewer: @JennyHajee 
```suggestion
|*Service Type* |*REST APIs* will be preselected.
```

> |*Authentication Method* |Select *Access key*.

Reviewer: @JennyHajee 
```suggestion
|*Authentication Method* |*Access key* will be preselected.
```
